### ISSUE TITLE
Cuda OOM in 1.53

### ISSUE THREAD

**Issue by lejunzhu** [LostRuins/koboldcpp#581]
I'm using WSL and CUBLAS. Using 1.52.2 I can offload 15 layers, but using 1.53 I can't offload even 1 layer.

The command I use:
```
python koboldcpp.py --threads 10 --port 7860 --host 127.0.0.1 --usecublas lowvram --smartcontext --nommap --gpulayers X --context 4096 --model yi-34b-chat.Q4_K_M.gguf
```

The log from 1.53 (fail):
```
python koboldcpp.py --threads 10 --port 7860 --host 127.0.0.1 --usecublas lowvram --smartcontext --nommap --gpulayers 1 --context 4096 --model yi-34b-chat.Q4_K_M.gguf
***
Welcome to KoboldCpp - Version 1.53
Attempting to use CuBLAS library for faster prompt ingestion. A compatible CuBLAS will be required.
Initializing dynamic library: koboldcpp_cublas.so
==========
Namespace(model='yi-34b-chat.Q4_K_M.gguf', model_param='yi-34b-chat.Q4_K_M.gguf', port=7860, port_param=5001, host='127.0.0.1', launch=False, lora=None, config=None, threads=10, blasthreads=10, highpriority=False, contextsize=4096, blasbatchsize=512, ropeconfig=[0.0, 10000.0], smartcontext=True, noshift=False, bantokens=None, forceversion=0, nommap=True, usemlock=False, noavx2=False, debugmode=0, skiplauncher=False, hordeconfig=None, noblas=False, useclblast=None, usecublas=['lowvram'], gpulayers=1, tensor_split=None, onready='', multiuser=0, remotetunnel=False, foreground=False, preloadstory='', quiet=False, ssl=None)

...

llm_load_print_meta: LF token         = 315 '<0x0A>'
llm_load_tensors: ggml ctx size       =    0.21 MiB
llm_load_tensors: using CUDA for GPU acceleration

CUDA error 2 at ggml-cuda.cu:9674: out of memory
current device: 0
GGML_ASSERT: ggml-cuda.cu:9674: !"CUDA error"
```

The log from 1.52.2 (success):
```
python koboldcpp.py --threads 10 --port 7860 --host 127.0.0.1 --usecublas lowvram --smartcontext --nommap --gpulayers 15 --context 4096 --model yi-34b-chat.Q4_K_M.gguf
***
Welcome to KoboldCpp - Version 1.52.2
Attempting to use CuBLAS library for faster prompt ingestion. A compatible CuBLAS will be required.
Initializing dynamic library: koboldcpp_cublas.so
==========
Namespace(model='yi-34b-chat.Q4_K_M.gguf', model_param='yi-34b-chat.Q4_K_M.gguf', port=7860, port_param=5001, host='127.0.0.1', launch=False, lora=None, config=None, threads=10, blasthreads=10, highpriority=False, contextsize=4096, blasbatchsize=512, ropeconfig=[0.0, 10000.0], smartcontext=True, noshift=False, bantokens=None, forceversion=0, nommap=True, usemlock=False, noavx2=False, debugmode=0, skiplauncher=False, hordeconfig=None, noblas=False, useclblast=None, usecublas=['lowvram'], gpulayers=15, tensor_split=None, onready='', multiuser=0, remotetunnel=False, foreground=False, preloadstory='', quiet=False)

...

llm_load_print_meta: LF token         = 315 '<0x0A>'
llm_load_tensors: ggml ctx size = 19700.45 MiB
WARNING: failed to allocate 19700.45 MB of pinned memory: out of memory
llm_load_tensors: using CUDA for GPU acceleration
llm_load_tensors: mem required  = 14794.00 MiB
llm_load_tensors: offloading 15 repeating layers to GPU
llm_load_tensors: offloaded 15/61 layers to GPU
llm_load_tensors: VRAM used: 4906.45 MiB
...................................................................................................
Automatic RoPE Scaling: Using model internal value.
llama_new_context_with_model: n_ctx      = 4176
llama_new_context_with_model: freq_base  = 5000000.0
llama_new_context_with_model: freq_scale = 1
llama_new_context_with_model: KV self size  =  978.75 MiB, K (f16):  489.38 MiB, V (f16):  489.38 MiB
llama_build_graph: non-view tensors processed: 1264/1264
llama_new_context_with_model: compute buffer total size = 510.10 MiB
WARNING: failed to allocate 506.91 MB of pinned memory: out of memory
llama_new_context_with_model: VRAM scratch buffer: 506.91 MiB
llama_new_context_with_model: total VRAM used: 5413.36 MiB (model: 4906.45 MiB, context: 506.91 MiB)
Load Model OK: True
```

**Comment by LostRuins** [comment]
Will be fixed shortly

**Comment by LostRuins** [comment]
The program will currently crash if mmap is disabled and pinned memory is unable to be allocated. mmap is fine. If you cannot use mmap, please use the previous version for now. This will be fixed in the next version.

**Comment by LostRuins** [comment]
This should be fixed in 1.54

**Comment by lejunzhu** [comment]
> This should be fixed in 1.54

Yes, the problem disappears after I upgrade to 1.54.
